admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -275,6 +276,13 @@ class AdminModuleController {
     });
   }
 
+  onModuleUpgraded(event) {
+    const self = this;
+    const moduleElement = $(event.detail[0]);
+
+    console.log(`Module ${moduleElement.data('tech-name')} upgraded`)
```

Comment:
Yes the console.log shouldn't stay here, I guess you can leave a small code as example but commented:
```suggestion
    // Do whatever you need after a successful upgrade action
    // const moduleElement = $(event.detail[0]);
    // const techName = moduleElement.data('tech-name');
```